### PR TITLE
Adding Issue templates

### PR DESCRIPTION
**Pull request by Mrinank-Bhowmick**
## Aim

**_With issue forms, we can create issue templates that have customizable web form fields. We can encourage contributors to include specific, structured information by using issue forms in this repository. Issue forms are written in YAML using the GitHub form schema._**

## Some screenshots

[](https://postimg.cc/0MLZK5pF)
</br>
[](https://postimg.cc/crGXsH18)

### CODE REVIEW

**Comment by marina-p** [comment]
Hello, @Mrinank-Bhowmick, thank you for contributing these!  

Reviewing now...

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+name: '🐞 Bug report'
+description: Let us know about functionality that is not working as expected
```
Comment (marina-p) [inline .github/ISSUE_TEMPLATE/bug-report.yml:2]: >Let us know about functionality  [](http://example.com/codeflow?start=13&length=32)

How about 'report that something is not working as expected'?

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+title: "Bug report: <short description>"
+labels: ["bug"]
```
Comment (marina-p) [inline .github/ISSUE_TEMPLATE/bug-report.yml:4]: Please remove the default label.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      value: |
+        Thank you for reporting the issue you are facing.
+        Please complete the below form to ensure we have the necessary details to assist you.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Provide a short description of the issue you are facing
+    validations:
+      required: true
```
Comment (marina-p) [inline .github/ISSUE_TEMPLATE/bug-report.yml:17]: >required: true [](http://example.com/codeflow?start=6&length=14)

Please make this optional (for situations where this is just a repeat of the title).

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      description: Provide a description of what you expect to happen
+    validations:
+      required: true
+  - type: textarea
+    id: actual
+    attributes:
+      label: Actual results
+      description: Provide a description of what actually happens
+    validations:
+      required: true
```
Comment (marina-p) [inline .github/ISSUE_TEMPLATE/bug-report.yml:38]: >required: true [](http://example.com/codeflow?start=6&length=14)

please make this optional

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+  - type: input
+    id: pythonversion
+    attributes:
+      label: python version
+      description: |
+        Provide the version of python you are using
+        Execute `python --version` or `python -V` in your terminal to identify your version
+    validations:
+      required: false
```
Comment (marina-p) [inline .github/ISSUE_TEMPLATE/bug-report.yml:48]: we should add one for dotnet version

(Feel free not to fix if you don't have time to do this, I will add this later)

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 6]
```diff
@@ -0,0 +1,40 @@
+name: '🚀 Feature request'
+description: Please let us know if there is anything we can do better
+title: "Feature request: <short description>"
+labels: ["enhancement"]
```
Comment (marina-p) [inline .github/ISSUE_TEMPLATE/feature-request.yml:4]: Please remove label

**Review by marina-p — DISMISSED** [review]
:clock1:

**Comment by Mrinank-Bhowmick** [comment]
> 🕐

Sure , I will make the required changes.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 7]
```diff
@@ -0,0 +1,68 @@
+name: '🐞 Bug report'
+description: Let us know about functionality that is not working as expected
+title: "Bug report: <short description>"
```
[the remark this answers is not in the document]
Comment (Mrinank-Bhowmick) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: Done 👍

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (Mrinank-Bhowmick) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: Done 👍

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+        - Linux
+        - Other
+    validations:
+      required: false
+  - type: textarea
+    id: info
+    attributes:
+      label: Additional Info
+      description: |
+        Provide any additional information that may help with the identification of the root cause of this issue validations
```
Comment (marina-p) [inline .github/ISSUE_TEMPLATE/bug-report.yml:66]: >validations [](http://example.com/codeflow?start=113&length=11)

This should be on the next line

**Review by marina-p — CHANGES_REQUESTED** [review]
:clock1:

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 8 above)
Comment (Mrinank-Bhowmick) [reply .github/ISSUE_TEMPLATE/bug-report.yml:66]: Can you edit this and merge? I have turned on to edit by the maintainers.

**Comment by Mrinank-Bhowmick** [comment]
> clock1

Can you please check it now?

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 8 above)
Comment (marina-p) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: Done

**Review by marina-p — APPROVED** [review]
:shipit:

**Pull request merged**